09_palindromes/palindromes.js

```diff
@@ -1,4 +1,20 @@
-const palindromes = function () {
+const palindromes = function (str) {
+
+    let cleanStr = str.replace(/\W/g, '');
+    let upperStr = cleanStr.toUpperCase()
```

Comment:
You can also just use the cleanStr, no need to create a new variable:

```
let cleanStr = str.replace(/\W/g, '');
cleanStr = cleanStr.toUpperCase()
```

Reply:
done